clients/java/src/test/java/marquez/client/models/JobTest.java

```diff
@@ -18,16 +18,23 @@
 import static marquez.client.models.ModelGenerator.newRun;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.UUID;
+import marquez.client.Utils;
 import org.junit.jupiter.api.Test;
 
 @org.junit.jupiter.api.Tag("UnitTests")
 public class JobTest {
-  private static final Job JOB = newJobWith(newRun());
```

Comment:
I would add a helper method `newJobWith(UUID, UUID)`, where `UUID` if for the runID and the current version you'd like to assert.